### PR TITLE
Upgrade the ingest api to 0.5.0

### PR DESCRIPTION
**Pull request by jhunzik**
Updates the Ingest API to the newest version and fixes the itests. This change was already heroed by @emmberk and myself.

### CODE REVIEW

File: src/main/java/com/connexta/ingest/controllers/IngestController.java [hunk 1]
```diff
@@ -33,19 +33,16 @@ public IngestController(IngestService ingestService) {
...
-    log.info(
-        "Ingest request received acceptVersion={}, fileName={}, fileSize={}, mediaType={}",
-        acceptVersion,
-        fileName,
-        fileSize,
-        mediaType);
+      String acceptVersion, MultipartFile file, String correlationId, MultipartFile metacard) {
+    String mediaType = file.getContentType();
+    String fileName = file.getOriginalFilename();
+    Long fileSize = file.getSize();
```
Comment (emmberk) [inline src/main/java/com/connexta/ingest/controllers/IngestController.java]: ✏️ Inline mediaType and fileSize

File: src/main/java/com/connexta/ingest/controllers/IngestController.java [hunk 2]
```diff
@@ -33,19 +33,16 @@ public IngestController(IngestService ingestService) {
...
+    String metacardName = metacard.getOriginalFilename();
+    log.info("Ingest request received fileName={}", fileName);
+    log.info("Ignoring attached metacard: {}", metacardName);
```
Comment (emmberk) [inline src/main/java/com/connexta/ingest/controllers/IngestController.java]: ✏️ I don't think the name of the metacard file is useful. `log.info("Ignoring attached metacard)` is good enough, I think.

File: src/test/java/com/connexta/ingest/IngestITests.java [hunk 3]
```diff
@@ -46,6 +46,7 @@
 public class IngestITests {
```
Comment (emmberk) [inline src/test/java/com/connexta/ingest/IngestITests.java]: ✏️ Add a test to check that the request is invalid (400) when there is no metacard

File: src/main/java/com/connexta/ingest/controllers/IngestController.java (hunk 2 above)
Comment (jhunzik) [reply src/main/java/com/connexta/ingest/controllers/IngestController.java]: I agree, we won't have this message up for more than a couple weeks anyway.

**Pull request merged**